spec/lib/debts/letter_downloader_spec.rb

```diff
@@ -24,24 +24,42 @@ def get_vbms_fixture(path)
   end
 
   describe '#get_letter' do
-    let(:request_name) { 'GetDocumentContent' }
-    let(:document_id) { '{93631483-E9F9-44AA-BB55-3552376400D8}' }
-    let(:request_args) { document_id }
-    let(:content) { File.read('spec/fixtures/pdf_fill/extras.pdf') }
-
     before do
-      expect(vbms_client).to receive(:send_request).with(
-        request_double
-      ).and_return(
-        OpenStruct.new(
-          document_id: document_id,
-          content: content
-        )
+      # rubocop:disable RSpec/SubjectStub
```

Comment:
did you try making `request_double` into a `list_request` and `letter_request` for each time of request you'd want to stub?  That way you don't have to stub the object under test.